modules/ossm-deploy-application-workloads-in-each-mesh.adoc

```diff
@@ -0,0 +1,206 @@
+// Module included in the following assemblies:
+// install/ossm-deploying-multiple-service-meshes-on-single-cluster.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-deploy-application-workloads-in-each-mesh_{context}"]
+= Deploy application workloads in each mesh
+:context: ossm-deploy-application-workloads-in-each-mesh
+
+This procedure demonstrates how to deploy application workloads in separate namespaces.
+
+Run the following command to check for existing {istio} instances.
```

Comment:
As mentioned in the other comment, it looks like the `oc get istios` command (line 11-14) was not moved with the rest of the content under "Deploy multiple control planes", and is out of place here.  There is a sentence in that paragraph "Run the following command to check for existing Istio instances", which this command should come after.